src/App.tsx

```diff
@@ -4,22 +4,25 @@ import { createGlobalStyle } from 'styled-components';
 import Theme from './Theme';
 import { Fonts } from './content';
 
-import { Header } from './Header';
-import Home from './Home';
-import About from './About';
-import Projects from './Projects';
-import Apply from './Apply';
-import Contact from './Contact';
-import { Footer } from './Footer';
+import { Header } from './partials/Header';
+import Home from './pages/home/Home';
+import About from './pages/about/About';
+import Projects from './pages/projects/Projects';
+import Apply from './pages/apply/Apply';
+import Contact from './pages/contact/Contact';
+import { Footer } from './partials/Footer';
 
-const routes = [Home, About, Projects, Apply, Contact];
+import { Teams } from './pages/team';
+
+const routes = [Home, About, Projects, Apply, Contact, Teams];
 
 const links = [
```

Comment:
oh wow i never noticed that. I don't know what it does. Do you @meerabal ? It kinda looks like it may have been used before the routes list or something